src/chapters/web-apis/studio.rst

```diff
@@ -0,0 +1,411 @@
+.. index:: ! headless API
+
+Studio: Consuming the Coding Events API With Postman
+====================================================
+
+The UI of a browser is designed to make simple ``GET`` requests for URLs entered into its address bar. This design works great for browsing sites, but 
+falls short when working with APIs. Anything beyond a ``GET`` request is difficult to send via a browser address bar alone. Think about what is needed to 
+create a new CodingEvent. This type of request contains a body. Our MVC application included a view to allow us to test inputs. Our API, however, is 
+**headless**. It does not contain the client-side form. In order to test how it handles requests then, we need a way to interact with the API server without 
+the browser. In this studio, we work with Postman to explore how APIs can be consumed.
+
+Setup
+-----
+
+Install Postman
+^^^^^^^^^^^^^^^
+
+If you haven't done so already, :ref:`install Postman <postman-installation>`.
+
+Fork and Clone the API Source Code
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With Postman installed, we're ready to fork and clone the `CodingEvent API <https://github.com/LaunchCodeEducation/coding-events-api/tree/1-sqlite>`__. 
+Although it is implemented differently from CodingEvents MVC, you will find that most of the features from the MVC application have been supported through 
+endpoints in the API.
+
+.. admonition:: Note
+
+   Our focus in this course is on operations and as such we will not be going into the development of the API. However, feel free to explore the source 
+   code if you are curious about the similarities and differences between the .NET MVC and API implementations.
```

Comment:
Start with these lines, then go into "With Postman, installed ..."
Delete "Let's begin by forking and cloning ..." sentence. 

I think that might flow a little better.